java/arcs/core/storage/StorageProxy.kt

```diff
@@ -467,12 +503,19 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T>(
     }
 
     private fun notifyUpdate(data: T) {
+        // If this isn't our first update and the data's hashCode is equivalent to the old data's
+        // hashCode, no need to send an update.
```

Comment:
Is this preventing some sort of redundant update issue we were seeing?

Reply:
Not one that has manifest as a real bug, but it does make it difficult to be predictable in tests if we don't maintain this.